client/Components/GuestDropdown.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styles from './GuestDropdown.css';
+import GuestItem from './GuestItem.jsx';
+
+class GuestDropdown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      adults: 1,
+      children: 0,
+      infants: 0,
+      expand: false, // false in production
+    };
+    this.expanded = this.expanded.bind(this);
+  }
+
+  clickHander(e) {
```

Comment:
You may want to remove this for now since you're not using it yet.

Reply:
I think I will.